cmd/openshift-install/create.go

```diff
@@ -198,6 +198,12 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 
 		err := runner(rootOpts.dir)
 		if err != nil {
+			// Attempt to gather logs if cluster asset creation failed
+			if cmd.Name() == "cluster" {
```

Comment:
We don't want to make this Azure-specific?  Or otherwise opt in only platforms/clusters for which we expect to need it?  I'm slightly (but not very) concerned about distracting noise on platforms where it is not helpful.

Reply:
More details in on the Azure use case in [rhbz#1779375](https://bugzilla.redhat.com/show_bug.cgi?id=1779375).

Reply:
I don't have the insight to know which platforms it makes sense for, I could restrict it to just baremetal and azure I guess. Couldn't any platform fail in the cluster asset but have gotten far enough to get a bootstrap VM up and running?

Reply:
we shuold move this to https://github.com/openshift/installer/pull/4020/files#diff-266f03122200ace2d21f2e449b6a5503R90 (the handler for create cluster command)

Reply:
The link you have doesn't work (even before I rebased), what do you mean by the handler? 

It is here because the fatal log below exits immediately, nothing else gets run.

Reply:
@abhinavdahiya Please see my question above

Reply:
It's a link to line 90 (presumably in this file)

Reply:
I see, I assumed it was pointing to the PostRun func definition, but moving this code there doesn't make sense to me. When cluster creation fails, logrus.Fatal causes us to exit immediately. We never return anywhere from here.